tfkbnufft/nufft/interp_functions.py

```
@@ -219,7 +219,7 @@ def _map_body(inputs):
             y = y_not_shifted * shift
         return y
 
-    y = tf.map_fn(_map_body, [x, tm, om], dtype=x.dtype)
+    y = tf.map_fn(_map_body, [x, tm, om], dtype=x.dtype, parallel_iterations=x.shape[0])
```

Although I think this is a good idea, this won't work (but silently fail) in graph mode because `x.shape[0]` will be `None`.

The default value for this is 10 (which will be applied right now with `x.shape[0]` being `None`) in graph mode, and in eager mode it will use the correct value.

I think let's keep this enhancement for another PR.

---

Cool.